source/locales.tex

```diff
@@ -1802,18 +1802,18 @@
       explicit codecvt(size_t refs = 0);
 
       result out(
-          stateT& state,
-          const internT* from, const internT* from_end, const internT*& from_next,
-                externT*   to,       externT*   to_end,       externT*&   to_next) const;
```

Comment:
@zygoloid Should we fix the parameters as well?  I would, but the current indentation does make the differences between these functions more obvious.